prime-angular-frontend/src/app/modules/adjudication/shared/components/review-status-content/review-status-content.component.ts

```diff
@@ -44,16 +47,21 @@ class Reason {
 export class ReviewStatusContentComponent implements OnInit, OnChanges {
   @Input() public enrollee: HttpEnrollee;
   @Input() public hideStatusHistory: boolean;
+  @Output() public route: EventEmitter<string | (string | number)[]>;
   public previousStatuses: Status[];
   public reasons: Reason[];
   private questions: { [key: number]: string } = selfDeclarationQuestions;
 
+  public AdjudicationRoutes = AdjudicationRoutes;
+  public DISPLAY_ID_OFFSET = DISPLAY_ID_OFFSET;
+
   constructor(
     private utilsService: UtilsService,
     private enrolmentResource: EnrolmentResource,
     private configPipe: ConfigCodePipe
   ) {
     this.hideStatusHistory = false;
+    this.route = new EventEmitter<string | (string | number)[]>();
```

Comment:
There is a typing for this because its used multiple times throughout the app, we could update the typing here to RoutePath
`import { RoutePath } from '@lib/utils/route-utils.class';`
`this.route = new EventEmitter<RoutePath>();`

Reply:
added RoutePath